EfEnumToLookup/LookupGenerator/SqlServerHandler.cs

```diff
@@ -106,17 +106,16 @@ private string PopulateLookup(LookupData lookup)
 			}
 
 			sql.AppendLine(string.Format(@"
-MERGE INTO [{0}] dst
-	USING #lookups src ON src.Id = dst.Id
-	WHEN MATCHED AND src.Name <> dst.Name collate Latin1_General_BIN2 THEN
-		UPDATE SET Name = src.Name
-	WHEN NOT MATCHED THEN
-		INSERT (Id, Name)
-		VALUES (src.Id, src.Name)
-	WHEN NOT MATCHED BY SOURCE THEN
-		DELETE
-;"
-				, TableName(lookup.Name)));
+				MERGE INTO [{1}].[{0}] dst
```

Comment:
this was intentionally not indented to avoid unnecessary whitespace in the generated output